helm/values-prod.yaml

```diff
@@ -112,9 +112,23 @@ microservice-chart:
     MONGO_WAITING_QUEUE_MS: "2000"
     MONGO_HEARTBEAT_FREQUENCY_MS: "5000"
     MONGO_DB_NAME: "ecommerce"
+    REDIS_PROTOCOL: "rediss"
+    REDIS_PORT: "6380"
+    REDIS_SSL_ENABLED: "true"
+    REDIS_SUB_CONN_MIN: "4"
+    REDIS_SUB_CONN_MAX: "36"
```

Comment:
have those values been tweaked someway or is just a payment-wallet-cdc porting?
Maybe put a TODO comment here to remember to tune up those variables after soak tests reports